spec/swagger.yaml

```diff
@@ -183,6 +183,11 @@ tags:
   - name: Tags
     description: |-
       Tags can help you manage your strings. The Tags API allows you to add/remove string tags, and search for tags in a project.
+
+  - name: Vendors
+    description: |-
+      Vendors API is to be used by LSPs to view work available in Smartling.
```

Comment:
The Vendors API is for Language Service Providers to view work that's available in Smartling.